controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -399,45 +420,56 @@ public String getScopeName() {
     }
 
     @Override
-    public CompletableFuture<TxnStatus> sealTransaction(final UUID txId, final boolean commit,
+    public CompletableFuture<Pair<TxnStatus, Integer>> sealTransaction(final UUID txId, final boolean commit,
```

Comment:
Could you clarify the difference between `sealTransaction` and `sealActiveTxn` below? Is it worth documenting the semantics around these calls?